README.md

```diff
@@ -286,6 +164,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job working on this milestone. When I click on your MIT license link, it redirects to an error 404 page. It not professional to push broken links to Github. Kindly fix this issue.
![Screen Shot 2023-02-28 at 6 18 39 AM](https://user-images.githubusercontent.com/62849872/221884257-eea3a916-31d5-4d83-8759-087c062d05d0.png)